src/components/IconBadge/IconBadge.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react"
+import RepeatOneRoundedIcon from "@mui/icons-material/RepeatOneRounded"
+import RepeatRoundedIcon from "@mui/icons-material/RepeatRounded"
+import {
+  BaseFemaleIcon,
+  BaseMaleIcon,
+  BodyShapeIcon,
+  EarringIcon,
+  EyebrowsIcon,
+  EyesIcon,
+  EyewearIcon,
+  FacialHairIcon,
+  FeetIcon,
+  HairIcon,
+  HandsWearIcon,
+  HatIcon,
+  HelmetIcon,
+  LowerBodyIcon,
+  MaskIcon,
+  MouthIcon,
+  PlacesIcon,
+  PropsIcon,
+  SkinIcon,
+  SmartWearableIcon,
+  SoundIcon,
+  SparklesIcon,
+  TiaraIcon,
+  TopHeadIcon,
+  UnisexIcon,
+  UpperBodyIcon,
+  UtilityIcon,
+} from "../Icon"
+import { IconBadgeIcon, IconBadgeProps } from "./IconBadge.types"
+import {
+  IconBadgeButtonContainer,
+  ImageContainer,
+  TextContainer,
+} from "./IconBadge.styled"
+
+const getIcon = (icon: IconBadgeIcon) => {
+  const color = "inherit"
+  switch (icon) {
+    case IconBadgeIcon.BaseFemale:
+      return <BaseFemaleIcon fontSize={"small"} color={color} />
```

Comment:
Are the SVGs affected by the font size?

Reply:
Material use the property `fontSize` to manage the size of the svg inside the `SvgIcon` component

[Here](https://mui.com/material-ui/api/svg-icon/) is more information about the SvgIcon API